HHS/OPRE-OPS
# Starts BL_ID Front End

## What changed
- replace Services Component short code for BL_ID
- refactors isDisabled helper
- adds disabled state for duplicate icon when not editable
- better confirmation modal text for deleting BLI
- adds hard delete to BLIs in all BLI list

## Issue

#2020 

## How to test

1. See Agreement BLIs and see IDs
2. GOTO Agreement wizard
3. add BLIS
4. see BLI_IDs in the table
5. delete an BLI
6. see BLI_ID in the confirmation modal

## Screenshots

<img width="1018" alt="image" src="https://github.com/HHS/OPRE-OPS/assets/4629398/d720dc0c-d706-4aa1-af07-f596e444b9de">

<img width="657" alt="image" src="https://github.com/HHS/OPRE-OPS/assets/4629398/dce61314-c698-4d33-833b-675f1b6dc82a">


## Definition of Done Checklist
